manager-bundle/src/HttpKernel/ContaoKernel.php

```diff
@@ -216,6 +216,10 @@ public function registerContainerConfiguration(LoaderInterface $loader): void
         if ($servicesFile = $this->getConfigFile('services.yml')) {
             $loader->load($servicesFile);
         }
+
+        if (is_dir($this->getProjectDir().'/src')) {
+            $loader->load(__DIR__.'/../Resources/config/autoload.php');
```

Comment:
As discussed in Mumble on January 16th, the file should be called `services.php` and go into the `src/Resources/skeleton/config/` folder.

Reply:
Changed in 12cd852